settings.gradle

```diff
@@ -24,4 +24,6 @@ include 'data-prepper-benchmarks:service-map-stateful-benchmarks'
 include 'data-prepper-plugins:otel-trace-raw-prepper'
 include 'data-prepper-plugins:otel-trace-source'
 include 'data-prepper-plugins:peer-forwarder'
+include 'data-prepper-plugins:blocking-buffer'
+findProject(':data-prepper-plugins:blocking-buffer')?.name = 'blocking-buffer'
```

Comment:
Is this needed?

Reply:
It was automatically added when the blocking-buffer subproject was created, I'll check if the findProject is needed or not.